gauntlet/packages/gauntlet-solana/src/commands/internal/solana.ts

```diff
@@ -81,18 +108,27 @@ export default abstract class SolanaCommand extends WriteCommand<TransactionResp
   signAndSendRawTx = async (
     rawTxs: TransactionInstruction[],
     extraSigners?: Keypair[],
-    overrides: {
-      units?: number
-      price?: number
-    } = {},
+    overrides: Overrides = {},
+    retryCount: number = 0,
   ): Promise<TransactionSignature> => {
     const { blockhash, lastValidBlockHeight } = await this.provider.connection.getLatestBlockhash()
+    if (this.flags.priorityFeesConstant && this.flags.priorityFeesHistorical) {
+      throw new Error('Cannot have both constant priority fees set and historical priority fees')
+    }
 
-    if (!overrides.units && !!this.flags.computeUnits) overrides.units = this.flags.computeUnits
-    if (!overrides.price && !!this.flags.computePrice) overrides.price = this.flags.computePrice
-
+    // Default send with priority fees found using block estimator utils if not set
+    if (!overrides.price && validateHistoricalPriorityFeeInput(this.flags.priorityFeesHistorical)) {
+      const [percentile, nBlocks] = this.flags.priorityFeesHistorical.split(',')
+      overrides.price = await this.calculatePriceFromHistoricalBlocks(nBlocks, percentile)
+      logger.info(
+        `Found ${percentile} percentile priority fees in past ${nBlocks} as: ${overrides.price}. Setting as Priority Fee`,
```

Comment:
you'd want to multiply percentile by 100. So instead of logging 0.5 pecentile it's actually the 50th percentile